test/test_worker.rb

```diff
@@ -18,7 +18,7 @@ def test_files_to_example_ids
       ]
     )
 
-    assert_equal expected, actual
+    assert_equal expected.sort, actual.sort
```

Comment:
I guess those `.sort`s can be part of the previous commit as well.

Reply:
Right!